src/Framework/ContaoFramework.php

```diff
@@ -187,7 +189,7 @@ private function setConstants()
         }
 
         // Define the login status constants in the back end (see #4099, #5279)
-        if (!$this->isFrontendScope()) {
+        if (!$this->request || !$this->isFrontendScope($this->request)) {
```

Comment:
`!$this->request` should not be necessary, because `isFrontendScope()` will handle `null`, right?